src/wp-admin/includes/list-table.php

```diff
@@ -58,6 +58,20 @@ function _get_list_table( $class_name, $args = array() ) {
 			$args['screen'] = null;
 		}
 
+		/**
+		 * Filters the list table class to instantiate.
+		 *
+		 * @since 6.1.0
+		 *
+		 * @param string $class_name The list table class to use.
+		 * @param array  $args       An array containing _get_list_table() arguments.
+		 */
+		$class_name = apply_filters( 'wp_list_table_class_name', $class_name, $args );
+
+		if ( ! is_string( $class_name ) || ! class_exists( $class_name ) ) {
+			return false;
+		}
```

Comment:
```suggestion
		$custom_class_name = apply_filters( 'wp_list_table_class_name', $class_name, $args );

		if ( is_string( $custom_class_name ) && class_exists( $custom_class_name ) ) {
			$class_name = $custom_class_name;
		}
```
@desrosj Do you mean something like this?

Reply:
Exactly! I haven't dug in to see how problematic returning `false` would be, but I assume it would potentially cause fatal errors. This would allow a list table class to go missing for some reason and still display the data in a default format.

Reply:
I think it's a good suggestion. The function can already return false so the use case should probably be checked but it isn't....

https://github.com/WordPress/wordpress-develop/blob/991dc2a5cc4d13a331f119e478dbd9745a7512d4/src/wp-admin/edit-comments.php#L19-L20

https://github.com/WordPress/wordpress-develop/blob/de3fbdecc0eca1ec876dfd04bbf642a7f60a57b4/src/wp-admin/edit-tags.php#L41-L42

etc, etc, etc